tools/PI/DevHome.PI/ViewModels/InsightsPageViewModel.cs

```diff
@@ -1,77 +1,21 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System.Collections.ObjectModel;
-using System.ComponentModel;
 using System.Diagnostics;
-using System.Linq;
 using CommunityToolkit.Mvvm.ComponentModel;
 using DevHome.Common.Extensions;
-using DevHome.PI.Models;
+using DevHome.PI.Services;
 using Microsoft.UI.Xaml;
 
 namespace DevHome.PI.ViewModels;
 
 public partial class InsightsPageViewModel : ObservableObject
 {
-    private readonly BarWindowViewModel _barWindowViewModel;
-    private Process? _targetProcess;
-
-    [ObservableProperty]
-    private ObservableCollection<Insight> _insightsList;
-
     [ObservableProperty]
```

Comment:
Same here. I don't think we're looking for changes to the actual PIInsightsService, only the collection it exposes, correct?

Reply:
In this case we need it, since we use ViewModel.InsightsService.InsightsList in the Insights page xaml file